pkg/pureutils/purefb.go

```diff
@@ -4,12 +4,13 @@ import (
 	"fmt"
 	"github.com/portworx/torpedo/drivers/pure/flashblade"
 	"regexp"
+	"strings"
 )
 
 // PureCreateClientAndConnect Create FB Client and Connect
 func PureCreateFbClientAndConnect(fbMgmtEndpoint string, apiToken string) (*flashblade.Client, error) {
-	fbClient, err := flashblade.NewClient(fbMgmtEndpoint, "", "", apiToken,
-		"", false, false, "", nil)
+	fbClient, err := flashblade.NewClient(fbMgmtEndpoint, apiToken, "", "",
```

Comment:
If one of these parameters is the user-agent, can we set it to be "Torpedo" or something similar so that we know where it's coming from? In case we overload the FA/FB REST server, we want to know who the culprit(s) are.